husky_description/urdf/dual_arm_husky.urdf.xacro

```diff
@@ -3036,7 +3036,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <!-- end of joint list -->
   <!-- joint attaching parent link to robotiq hand -->
   <joint name="r_robotiq_hand_joint" type="fixed">
-    <origin rpy="1.57 0 0" xyz="0 0 0.05398"/>
+    <origin rpy="1.57 0 2.355" xyz="0 0 0.05398"/>
```

Comment:
@TheDash We've approximately guessed these rotation values from how the grippers were oriented in visualisation and in reality. Are there any documents on how they should have been mounted during installation? Is such a variation in rotation possible or are there physical alignment points? Both hands are "off" to varying degrees